### PR TITLE
[Not merge - To be tested] - Ping-Pong with pending messages #issue 122

### PR DESCRIPTION
**Pull request by frsela**

### CODE REVIEW

**Comment by willyaranda** [comment]
@acperez , please check this, do you like it?

File: src/ns_ua/ws_server.js [hunk 1]
```diff
@@ -305,6 +302,29 @@ server.prototype = {
...
         switch(query.messageType) {
+          case "PING":
+            log.debug("WS::onWSMessage --> UA PING message");
+            dataManager.getNodeData(connection.uatoken, function(error, data) {
+              if(error) {
+                log.debug("WS::onWSMessage --> Failing recovering UA");
+                connection.res({
+                  errorcode: errorcodesWS.UATOKEN_NOT_FOUND,
+                  extradata: { messageType: "PONG" }
+                });
```
Comment (willyaranda) [inline src/ns_ua/ws_server.js:313]: In this case, we should disconnect the client.

Comment (frsela) [inline src/ns_ua/ws_server.js:313]: agree

**Comment by willyaranda** [comment]
@acperez ping here, it's useful?

**Comment by frsela** [comment]
Also related to #175

**Comment by frsela** [comment]
Working on a new simpler PING/PONG version. This patch is obsolete

**Pull request closed without merging**